components/infobox/commons/infobox_league.lua

```diff
@@ -0,0 +1,245 @@
+local Class = require('Module:Class')
+local Cell = require('Module:Infobox/Cell')
+local Infobox = require('Module:Infobox')
+local Template = require('Module:Template')
+local Table = require('Module:Table')
+local Namespace = require('Module:Namespace')
+local String = require('Module:String')
+local Links = require('Module:Links')
+
+local getArgs = require('Module:Arguments').getArgs
+
+local League = Class.new()
+local Language = mw.language.new('en')
+
+function League.run(frame)
+    return League:createInfobox(frame)
+end
+
+function League:createInfobox(frame)
+    local args = getArgs(frame)
+    self.frame = frame
+    self.pagename = mw.title.getCurrentTitle().title
+    self.name = args.name or self.pagename
+
+    if args.wiki == nil then
+        return error('Please provide a wiki!')
+    end
+
+    local infobox = Infobox:create(frame, args.wiki)
+
+    infobox :name(args.name)
+            :image(args.image, args.default)
+            :centeredCell(args.caption)
+            :header('League Information', true)
+            :fcell(Cell :new('Series')
+                        :options({})
+                        :content(
+                            League:_createSeries(frame, args.series, args.abbrevation),
+                            League:_createSeries(frame, args.series2, args.abbrevation2)
+                        )
+                        :make()
+            )
+            :fcell(Cell  :new('Organizer')
+                        :options({})
+                        :content(
+                            League:_createOrganizer(
+								args.organizer, args['organizer-name'], args['organizer-link'], args.organizerref),
+                            League:_createOrganizer(
+								args.organizer2, args['organizer-name2'], args['organizer-link2'], args.organizerref2),
+                            League:_createOrganizer(
+								args.organizer3, args['organizer-name3'], args['organizer-link3'], args.organizerref3),
+                            League:_createOrganizer(
+								args.organizer4, args['organizer-name4'], args['organizer-link4'], args.organizerref4)
```

Comment:
```suggestion
                            League:_createOrganizer(
								args.organizer4, args['organizer-name4'], args['organizer-link4'], args.organizerref4)
                            League:_createOrganizer(
								args.organizer5, args['organizer-name5'], args['organizer-link5'], args.organizerref5)
```

Reply:
Not sure what is happening here

Reply:
adding a 5th organizer

Reply:
Went a slightly different route, deduplicated code and made the limit infinite.